api_backend.yaml

```diff
@@ -786,6 +791,52 @@ definitions:
     $ref: "https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v0.8.6/api_pagopa.yaml#/definitions/SpezzoniCausaleVersamento"
   SpezzoniCausaleVersamentoItem:
     $ref: "https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v0.8.6/api_pagopa.yaml#/definitions/SpezzoniCausaleVersamentoItem"
+  
+  CreatedMessageWithContentAndAttachments:
+    type: object
+    properties:
```

Comment:
can you try using allOf with `CreatedMessageWithoutContentAndAttachments` +
`MessageContentWithAttachments` ?